backend/views/game.ejs

```diff
@@ -0,0 +1,69 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Uno Game</title>
+    <link rel="stylesheet" type="text/css" href="/styles/game.css">
+</head>
+<body>
+  <div class="players-container">
+    <% playerList.forEach(player => { %>
+        <div class="opponent-container">
+            <p class="opponent-name">
+                <%= player.name %>
+            </p>
+            <div class="opponent-container-bottom">
+                <img src="/images/cards/uno_back.png" alt="back of uno card" />
+                <p class="opponent-hand-size">
+                    <%= player.handSize + "X" %>
+                </p>
+            </div>
+        </div>
+        
+    <% }); %>
+  </div>
+  <main>
+    <div class="deck-container">
+        <img src="/images/cards/uno_back.png" alt="back of uno card" />
+       
+    </div>
+    <div class="discard-container">
+        <img src="<%= '/images/cards/' + discardCard.color + '_' + discardCard.symbol + '.png' %>" alt="<%= discardCard.color + ' ' + discardCard.symbol %>" />
+    </div>
+    <div class="game-buttons">
+        <form action="/game" method="POST">
+            <button type="submit" name="draw" value="draw">Draw</button>
+            <button type="submit" name="uno" value="uno">Uno</button>
+            <button name="play-button" id="play-button">Play</button>
+        </form>
+       
+    </div>
+    <div id="chat-container">
+        <div id="chat-messages">
+            <% (chatMessages?.length > 0) ? chatMessages.forEach(message => { %>
+                <p class="chat-message">
+                    <%= message %>
+                </p>
+            <% }) : null %>
+        </div>
+        <form action="/game/sendMessage" method="POST">
+            <input type="text" name="message" placeholder="Type a message..." />
+            <button type="submit" name="sendMessage" value="send">Send</button>
+        </form>
+    </div>
```

Comment:
Can you make this a partials? The authorized homepage, lobby, and game pages will all need a chat box so having this as a partial would be super helpful